web/hooks/use-feed.ts

```diff
@@ -20,6 +27,12 @@ export const useFeed = (
   }
 ) => {
   const { topic } = options ?? {}
+
+  const [boosts, setBoosts] = useState<BoostsType>()
+  useEffect(() => {
+    if (user) getBoosts(user.id).then(setBoosts as any)
```

Comment:
Seems like we'll occasionally get errors when the ad runs out of funds (if someone else claims the last mana after you fetched the boost).
Any easy ideas to address this?

Reply:
some options:
- require an extra 10 mana on boosting a new market as a "flat fee", and the query for ads is instead is `market_ads.funds > 10`. The fee usually will end up as "dust" in the ad, but occasionally it will be used to pay out both sides in a race
- pay out anyways so long as funds > -10, and we eat the cost
- subscribe to txns and hide dead ads (seems annoying for now)

Reply:
Maybe if there's x mana of funds left, where x < 10, there's only an x% chance that it would show in your feed.